server/adapters/logAdapter.js

```diff
@@ -0,0 +1,70 @@
+//
+// Adapter Log
+//
+module.exports = function initLogAdapter(models, adapters) {
+
+  //
+  // Initialize Bunyan logger and store it in libs
+  // @TODO: Fix model dependency
+  //
+  adapters.createLogger = function (libs) {
+
+    const Log = models.log;
+
+    const bunyanMongodbStream = require('bunyan-mongodb-stream')({ model: Log });
```

Comment:
Yes, external libraries should have their own import and initialization file under `libs/`